IonDotnet/Internals/Text/TextScanner.cs

```diff
@@ -1449,6 +1458,20 @@ private void UnreadChar(int c)
             }
         }
 
+        /// <summary>
+        /// Any CLOB with double quote, can hold only one " " and the only
+        /// acceptable character after the second double quote is }
+        /// </summary>
+        private void HandleClobsWithDoubleQuote()
+        {
+            var c = SkipOverWhiteSpace(CommentStrategy.Error);
+            if (c != '}')
+            {
+                throw new IonException($"Invalid character format: {(char)c}");
```

Comment:
I recommend using the same error message as on line 1929.

Reply:
Updated the message